src/node/gov/handlers/recovery.h

```diff
@@ -195,7 +205,7 @@ namespace ccf::gov::endpoints
         "/recovery/members/{memberId}:recover",
         HTTP_POST,
         api_version_adapter(submit_recovery_share),
-        ccf::no_auth_required)
+        detail::active_member_sig_only_policies("recovery_share"))
```

Comment:
Don't know enough to know if this is the correct policy/policy inputs, so just leaving a note for you to double check it :smile:

Reply:
It is correct (active members through COSE Sign1), and consistent with other POST endpoints: https://github.com/microsoft/CCF/blob/7372e76daf0d21d540ac6b6a342053c43a666adf/src/node/gov/handlers/helpers.h#L18

always good to check :)